docs/sources/live/configure-grafana-live.md

```diff
@@ -19,6 +19,14 @@ The number of maximum WebSocket connections users can establish with Grafana is
 
 In case you want to increase this limit, ensure that your server and infrastructure allow handling more connections. The following sections discuss several common problems which could happen when managing persistent connections, in particular WebSocket connections.
 
+## Request origin check
+
+To avoid hijacking WebSocket connections Grafana Live checks Origin request header sent by a client in HTTP Upgrade request. Requests without Origin header set pass through without any origin check.
+
```

Comment:
```suggestion
To avoid hijacking of WebSocket connection Grafana Live checks the Origin request header sent by a client in an HTTP Upgrade request. Requests without Origin header pass through without any origin check.
		

```